### ISSUE TITLE
Intake up/down and open/close in parallel

### ISSUE THREAD

**Issue by MatthewSpofford** [FRC4048/FRC2018_Java#71]
Test that we can make that happen (not sure because both require intake)

**Comment by snikopoulos** [comment]
I think we will need to have a single command that makes both operations. It would combine 
GripIntake and RaiseIntake.

**Comment by MatthewSpofford** [comment]
this was tested in toggle() but not in getCube()

**Comment by batchen1** [comment]
I think this isn't working because the 2 commands require the intake.
We can split it into 2 subsystems, or just run sequentially.

**Comment by batchen1** [comment]
Decided to run them sequentially
